package.json

```diff
@@ -44,12 +44,12 @@
     "core-js": "^3.29.0",
     "graphql": "^16.6.0",
     "lodash": "^4.17.21",
-    "next": "13.3.0",
+    "next": "^13.4.7",
```

Comment:
Have you followed the migration guide for this? 
https://github.com/flexbox/weshipit.today/blob/main/docs/nx-migration.md